src/test/java/com/softserve/academy/spaced/repetition/controller/CourseControllerTest.java

```diff
@@ -34,19 +39,25 @@
     @Mock
     private CourseService courseService;
 
+    final int NUMBER_PAGE = 1;
+    final String SORT_BY = "name";
+    final boolean ASCENDING = true;
```

Comment:
Just remove it.

There is Boolean.TRUE if you really need it.

Reply:
I should delete only boolean value? Or all constants?

Reply:
There is no need to define a boolean constant as it is already defined in java.lang.Boolean class.